documentation/modules/ROOT/partials/modules/all-connectors/proc-running-an-ad-hoc-snapshot-with-additional-conditions.adoc

```diff
@@ -0,0 +1,78 @@
+If you want a snapshot to include only a subset of the content in a {data-collection}, you can modify the signal request by appending an `additional-conditions` parameter to the snapshot signal.
+
+The SQL query for a typical snapshot takes the following form:
+
+[source,sql,subs="+attributes,+quotes"]
+----
+SELECT * FROM _<tableName>_ ....
+----
+
+By adding an `additional-conditions` parameter, you append a `WHERE` condition to the SQL query, as in the following example:
+
+[source,sql,subs="+attributes,+quotes"]
+----
+SELECT * FROM _<data-collection>_ WHERE _<filter>_ ....
+----
+
+The following example shows a SQL query to send an ad hoc incremental snapshot request with an additional condition to the signaling {data-collection}:
+[source,sql,indent=0,subs="+attributes,+quotes"]
+----
+INSERT INTO _<signalTable>_ (id, type, data) VALUES (_'<id>'_, _'<snapshotType>'_, '{"data-collections": ["_<tableName>_","_<tableName>_"],"type":"_<snapshotType>_","additional-conditions":[{"data-collection": "_<tableName>_", "filter": "_<additional-condition>_"}]}');
```

Comment:
Should this example also make use of the `collection.container` variable?

Reply:
I think that if you read it in context, it becomes more clear that the example that you reference shows the general format for a query, using `signalTable` and `tableName` variables (as well as `snapshotType` and `additional-condition`) in place of actual values. The followup examples then show an actual query that replaces the variables  with dummy values, such as `myschema.debezium_signal` and `"schema1.table1", "schema1.table2"`. This mirrors the [approach that's used in the published documentation](https://debezium.io/documentation/reference/2.7/connectors/db2.html#db2-incremental-snapshots-additional-conditions). 
I suppose that I could make it more clear by explicitly stating up front that the first example shows the general format, and maybe making the variable names more descriptive (e.g., `fullyQualifiedTableName`).  WDYT?

Reply:
In that case, feel free to merge yourself.